benchflow-driver-maker/api/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
I think we can cover this by .gitignore in the root of the repo or project, so that we don't have so many .gitignore files.